binding.cc

```diff
@@ -37,6 +39,8 @@ struct worker_s {
   std::string last_exception;
   Persistent<Function> recv;
   Persistent<Context> context;
+  std::map<std::string, Eternal<Module>> modules;
```

Comment:
Should be an `unordered_map` (O(1) instead of O(log N)) unless you know of a reason we need it to be ordered.

Same applies for `identifiers`